packages/salesforcedx-utils/test/jest/cli/__snapshots__/command.test.ts.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
```

Comment:
our first snapshot.  These are generated by jest the first time a `shouldMatchSnapshot` type call is made. Then after the value found here is compare on the matchSnapshot call in future tests.  If it doesn't match the assert will fail. If the value has change you can updated it with a flag passed to the test (included in the failure output). 

Thing to take away is snapshot should be treated as code and reviewed closely on change